scripts/generate_schemas.py

```diff
@@ -1,102 +0,0 @@
-#!/usr/bin/env python
```

Comment:
I'm guessing this is no longer used?

Reply:
I don't know why it's still here. So I deleted it. It was used for combining yaml files AFAICS, which we're no longer doing in favour of jsonnet.